src/main/java/com/locat/api/domain/user/entity/PlatformType.java

```diff
@@ -0,0 +1,18 @@
+package com.locat.api.domain.user.entity;
+
+public enum PlatformType {
+    IOS("ios"),
+    ANDROID("android");
+
+    private final String value;
+
+    PlatformType(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+
+}
```

Comment:
lombok을 활용하면 더 간결하게 작성할 수 있을 것 같아요!

```suggestion
@Getter
@AllArgsConstructor
public enum PlatformType {
    IOS("ios"),
    ANDROID("android");

    private final String value;

}